streamer/packager_node.py

```diff
@@ -141,6 +141,9 @@ def _setup_stream(self, stream: OutputStream) -> str:
     if stream.type == MediaType.AUDIO:
       dict['hls_group_id'] = str(cast(AudioCodec, stream.codec).value)
 
+    if stream.type == MediaType.VIDEO and self._pipeline_config.generate_iframe_playlist:
+      dict['iframe_playlist_name'] =  stream.get_identification() + '.m3u8'
```

Comment:
can we also have this outputing to an `iframes/` folder or even prepend the file name with `iframe_`.

we use `hls.m3u8` for the master playlist (variable)
`dash.mpd` for dash manifest (variable)
`stream_xx.m3u8` for m3u8 playlists
`video_360p_210k_av1_2.mp4` and such for the media files. the iframe files will thus get in between those.

Reply:
Done --> https://github.com/shaka-project/shaka-streamer/pull/205/commits/ad675b0032386baae5328d83f7add69d8d333243